cmd/kube-apiserver/app/options/options_test.go

```diff
@@ -303,12 +304,15 @@ func TestAddFlags(t *testing.T) {
 		EgressSelector: &apiserveroptions.EgressSelectorOptions{
 			ConfigFile: "/var/run/kubernetes/egress-selector/connectivity.yaml",
 		},
-		EnableLogsHandler:                 false,
-		EnableAggregatorRouting:           true,
-		ProxyClientKeyFile:                "/var/run/kubernetes/proxy.key",
-		ProxyClientCertFile:               "/var/run/kubernetes/proxy.crt",
-		Metrics:                           &metrics.Options{},
-		Logs:                              logs.NewOptions(),
+		EnableLogsHandler:       false,
+		EnableAggregatorRouting: true,
+		ProxyClientKeyFile:      "/var/run/kubernetes/proxy.key",
+		ProxyClientCertFile:     "/var/run/kubernetes/proxy.crt",
+		Metrics:                 &metrics.Options{},
+		Logs:                    logs.NewOptions(),
+		OpenTelemetry: &apiserveroptions.OpenTelemetryOptions{
+			ConfigFile: "/var/run/kubernetes/opentelemetry_config.yaml",
```

Comment:
what's the value of a default path? I would prefer to see this being disabled by default.

Reply:
It is disabled by default.  This test is checking that the `--opentelemetry-config-file=/var/run/kubernetes/opentelemetry_config.yaml` flag above is translated into the correct OpenTelemetryOptions struct here.